lm_example.jsonl

```diff
@@ -0,0 +1,10 @@
+{"source": "The present invention provides compounds suitable for use in the treatment of conditions where it is beneficial to halt bone loss and kill cancer cells, particularly in metastases to and primary tumours in the bone and surrounding tissues.", "target": "Consequently the present invention provides compounds comprising a bisphosphonate moiety linked to a phytochemical, pharmaceutical compositions thereof and methods of treatment of bone diseases and/or proliferative disorders.", "claim": "A compound of formula Q-T-L wherein Q is a bisphosphonate moiety, T is linker and L is an anti-osteolytic or osteoinductive phytochemical."}
```

Comment:
I would remove this file, keeping only the one inside `resources`

Reply:
Done